README.md

```diff
@@ -5,6 +5,14 @@ Redis Output Plugin For fluentd
 [Fluentd]: http://fluentd.org/
 [Redis]: http://redis.io/
 
+Requirements
+------------
+
+| fluent-plugin-redis-store | fluentd | ruby |
+|------------------------|---------|------|
+| >= 0.2.0 | >= v0.14.0 | >= 2.1 |
```

Comment:
You have set the minimum version as `0.14.15` in `fluent-plugin-redis-store.gemspec`.
https://github.com/pokehanai/fluent-plugin-redis-store/blob/master/fluent-plugin-redis-store.gemspec#L19
So I want to have this line `>= v0.14.15`.